staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/v2/conversion_test.go

```diff
@@ -819,62 +819,62 @@ func fuzzFuncs(f *fuzz.Fuzzer, refFunc func(ref *spec.Ref, c fuzz.Continue, visi
 		}
 	}
 	f.Funcs(
-		func(ref *spec.Ref, c fuzz.Continue) {
+		func(ref *spec.Ref, c randfill.Continue) {
 			refFunc(ref, c, invisible == 0)
 		},
-		func(sa *spec.SchemaOrStringArray, c fuzz.Continue) {
+		func(sa *spec.SchemaOrStringArray, c randfill.Continue) {
 			*sa = spec.SchemaOrStringArray{}
-			if c.RandBool() {
-				c.Fuzz(&sa.Schema)
+			if c.Bool() {
+				c.Fill(&sa.Schema)
 			} else {
-				c.Fuzz(&sa.Property)
+				c.Fill(&sa.Property)
 			}
 			if sa.Schema == nil && len(sa.Property) == 0 {
 				*sa = spec.SchemaOrStringArray{Schema: &spec.Schema{}}
 			}
 		},
-		func(url *spec.SchemaURL, c fuzz.Continue) {
+		func(url *spec.SchemaURL, c randfill.Continue) {
 			*url = spec.SchemaURL("http://url")
 		},
-		func(s *spec.Dependencies, c fuzz.Continue) {
+		func(s *spec.Dependencies, c randfill.Continue) {
 			enter(s, false, c)
 			defer leave(false)
 
 			// and nothing with invisible==false
 		},
-		func(p *spec.SimpleSchema, c fuzz.Continue) {
-			// gofuzz is broken and calls this even for *SimpleSchema fields, ignoring NilChance, leading to infinite recursion
+		func(p *spec.SimpleSchema, c randfill.Continue) {
+			// randfill is broken and calls this even for *SimpleSchema fields, ignoring NilChance, leading to infinite recursion
```

Comment:
Should we file a bug against randfill and try to fix that?

Reply:
We could -- this isn't NEW though. Opened an issue

Reply:
Agreed. It just seemed like a good time to take note of where we are with randfill because these TODOs will never be looked at again once this PR is merged. Same with the other one.